src/Controller/DefaultController.php

```diff
@@ -31,33 +33,70 @@ class DefaultController extends AbstractController
      */
     public function nasIndex(FundingOrgFilter $fundingOrgFilter)
     {
-        $filter = array();
+        $fundingCycleList = array();
         if ($fundingOrgFilter->isActive()) {
```

Comment:
This works great, but what do you do if the filter is NOT active?

Reply:
So, by default should i always get NAS/GRP funding org data? because its anyways inside the grp-home default route

Reply:
I don't think we can make that assumption. Things may change somewhere else, and it should work either way, even tho this will only be used here.